app/views/ProfileView/index.js

```diff
@@ -81,7 +81,7 @@ class ProfileView extends React.Component {
 
 	componentWillReceiveProps(nextProps) {
 		const { user } = this.props;
-		if (user !== nextProps.user) {
+		if (equal(user, nextProps.user)) {
```

Comment:
`if (!equal(user, nextProps.user))`

Reply:
argh .... have pushed another commit. Thank you